### PR TITLE
Add CosmonovaRnD recipes

### PR DESCRIPTION
**Pull request by Besedin86**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: cosmonova-rnd/auth/0.7/config/packages/cosmonova_rnd_auth.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+doctrine:
```
Comment (Nyholm) [inline cosmonova-rnd/auth/0.7/config/packages/cosmonova_rnd_auth.yaml]: What if the user has already configured doctrine? (which would most likely be the case). 

It is good practice not touch other bundles configuration. Please remove `doctrine`, `framework` and `security`.

Comment (Besedin86) [reply cosmonova-rnd/auth/0.7/config/packages/cosmonova_rnd_auth.yaml]: This config just extend base config, not replace it. I know, It looks not good, but another way you should write more settings manually.

Comment (Nyholm) [reply cosmonova-rnd/auth/0.7/config/packages/cosmonova_rnd_auth.yaml]: Yeah, I would prefer you add a post-install comment and explain the manual steps.

File: cosmonova-rnd/jwt/0.7/config/packages/cosmonova_rnd_jwt.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+services:
+    CosmonovaRnD\JWT\Parser\Parser:
+        class: 'CosmonovaRnD\JWT\Parser\Parser'
```
Comment (Nyholm) [inline cosmonova-rnd/jwt/0.7/config/packages/cosmonova_rnd_jwt.yaml]: The "class" is not needed.

File: cosmonova-rnd/jwt/0.7/config/packages/cosmonova_rnd_jwt.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        class: 'CosmonovaRnD\JWT\Key\SignKey'
+        arguments: ['%env(file:JWT_PRIVATE_KEY)%','%env(file:JWT_PASSPHRASE)%']
+        autowire: true
+    CosmonovaRnD\JWT\Verifier\Verifier:
+        class: 'CosmonovaRnD\JWT\Verifier\Verifier'
+        autowire: true
+    CosmonovaRnD\JWT\Key\VerifyKey:
+        class: 'CosmonovaRnD\JWT\Key\VerifyKey'
+        arguments: ['%env(file:JWT_PUBLIC_KEY)%','%env(file:JWT_PASSPHRASE)%']
+        autowire: true
```
Comment (Nyholm) [inline cosmonova-rnd/jwt/0.7/config/packages/cosmonova_rnd_jwt.yaml]: What is being auto wired here?

File: cosmonova-rnd/jwt/0.7/config/packages/cosmonova_rnd_jwt.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    CosmonovaRnD\JWT\Signer\SignerFactory:
+        class: 'CosmonovaRnD\JWT\Signer\SignerFactory'
+        autowire: true
+    CosmonovaRnD\JWT\Signer\Signer:
+        class: 'CosmonovaRnD\JWT\Signer\Signer'
+        autowire: true
+    CosmonovaRnD\JWT\Key\SignKey:
+        class: 'CosmonovaRnD\JWT\Key\SignKey'
+        arguments: ['%env(file:JWT_PRIVATE_KEY)%','%env(file:JWT_PASSPHRASE)%']
+        autowire: true
```
Comment (Nyholm) [inline cosmonova-rnd/jwt/0.7/config/packages/cosmonova_rnd_jwt.yaml]: What is being auto wired here?

File: cosmonova-rnd/auth/0.7/config/packages/cosmonova_rnd_auth.yaml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        class: 'CosmonovaRnD\Auth\Http\TokenLoader'
+        arguments:
+            $tokenUri: '%env(OAUTH_TOKEN_URI)%'
+            $clientId: '%env(OAUTH_CLIENT_ID)%'
+            $clientSecret: '%env(OAUTH_CLIENT_SECRET)%'
+        autowire: true
+
+    CosmonovaRnD\Auth\Controller\AuthController:
+        class: 'CosmonovaRnD\Auth\Controller\AuthController'
+        autowire: true
```
Comment (Nyholm) [inline cosmonova-rnd/auth/0.7/config/packages/cosmonova_rnd_auth.yaml]: What you do autowire here?

File: cosmonova-rnd/auth/0.7/config/packages/cosmonova_rnd_auth.yaml [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+        autoconfigure: true
+
+    CosmonovaRnD\Auth\Http\TokenLoader:
+        class: 'CosmonovaRnD\Auth\Http\TokenLoader'
+        arguments:
+            $tokenUri: '%env(OAUTH_TOKEN_URI)%'
+            $clientId: '%env(OAUTH_CLIENT_ID)%'
+            $clientSecret: '%env(OAUTH_CLIENT_SECRET)%'
+        autowire: true
```
Comment (Nyholm) [inline cosmonova-rnd/auth/0.7/config/packages/cosmonova_rnd_auth.yaml]: What you do autowire here?

File: cosmonova-rnd/auth/0.7/config/packages/cosmonova_rnd_auth.yaml [hunk 7]
```diff
@@ -0,0 +1,26 @@
+services:
+    CosmonovaRnD\Auth\:
+        resource: '../../vendor/cosmonova-rnd/auth/src/*'
+        exclude: '../../vendor/cosmonova-rnd/auth/src/{Entity,Helper,Exception}'
+        autowire: true
+        autoconfigure: true
+
+    CosmonovaRnD\Auth\Http\TokenLoader:
+        class: 'CosmonovaRnD\Auth\Http\TokenLoader'
```
Comment (Nyholm) [inline cosmonova-rnd/auth/0.7/config/packages/cosmonova_rnd_auth.yaml]: Class is not needed.

File: cosmonova-rnd/auth/0.7/config/packages/cosmonova_rnd_auth.yaml (hunk 6 above)
Comment (Besedin86) [reply cosmonova-rnd/auth/0.7/config/packages/cosmonova_rnd_auth.yaml]: RouterInterface & LoggerInterface

File: cosmonova-rnd/jwt/0.7/config/packages/cosmonova_rnd_jwt.yaml [hunk 8]
```diff
@@ -0,0 +1,13 @@
+services:
+    CosmonovaRnD\JWT\Parser\Parser:
+        autowire: true
+    CosmonovaRnD\JWT\Signer\SignerFactory:
```
Comment (Nyholm) [inline cosmonova-rnd/jwt/0.7/config/packages/cosmonova_rnd_jwt.yaml]: Isnt this requiring a string?

File: cosmonova-rnd/jwt/0.7/config/packages/cosmonova_rnd_jwt.yaml [hunk 9]
```diff
@@ -0,0 +1,13 @@
+services:
+    CosmonovaRnD\JWT\Parser\Parser:
+        autowire: true
```
Comment (Nyholm) [inline cosmonova-rnd/jwt/0.7/config/packages/cosmonova_rnd_jwt.yaml]: What is autowired here?

You could just write
```
CosmonovaRnD\JWT\Parser\Parser: ~
```

**Comment by Besedin86** [comment]
I'd rework my package structures. So I'll close this pull request and open another one later.

**Pull request closed without merging**